exercises/grep/description.md

```diff
@@ -1,77 +1,25 @@
-# Description
+# Instructions
 
-Search a file for lines matching a regular expression pattern. Return the line
-number and contents of each matching line.
+Search files for lines matching a search string and return all matching lines.
 
-The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command can be used to search for lines in one or more files
-that match a user-provided search query (known as the *pattern*).
+The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command searches for lines in files that match a regular expression. Your task is to implement a simplified `grep` function, which supports searching for literal strings.
```

Comment:
Can you move the second sentence (`Your task`) to the next line?

As we're updating this file, it should now follow [the Exercism Markdown style](https://exercism.org/docs/building/markdown/markdown#layout).

Reply:
Done.